src/scaife-widgets/components/TextWidth.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <span
+    :class="['text-width-control', { active }]"
+    @click.prevent="$emit('input', width)"
+    >{{ text }}</span
+  >
+</template>
+
+<script>
+  export default {
+    props: ['value', 'width'],
+    computed: {
+      active() {
+        return this.value === this.width;
+      },
+      text() {
+        return this.width.charAt(0).toUpperCase() + this.width.slice(1);
+      },
+    },
+  };
+</script>
+
+<style lang="scss">
+  @import '../_variables.scss';
+  .text-width-control {
+    font-family: 'Noto Serif';
```

Comment:
Let's not define font-family on any widget.  If anything, we should define with a variable that can be set at the site level.